src/app/components/source-graphs/source-tooltip/source-tooltip.component.ts

```diff
@@ -44,6 +44,10 @@ export class SourceTooltipComponent implements AfterViewInit {
 
   constructor(private element: ElementRef) {}
 
+  public get height(): number {
```

Comment:
because the height doesn't change we can set it in `ngAfterViewInit`, to avoid the constant check